packages/commonwealth/client/scripts/views/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -65,6 +70,25 @@ export const Breadcrumbs = () => {
     currentDiscussion,
   );
 
+  useEffect(() => {
+    if (
+      topics &&
+      pathnames &&
+      topics.length > 0 &&
+      pathnames.length > 0 &&
+      pathnames[0].label === 'Discussions' &&
+      pathnames[1].label !== 'Overview' &&
+      pathnames[1].label !== 'archived'
+    ) {
+      const validTopics = topics.some(
+        (topic) => topic.name === pathnames[1].label,
+      );
+      if (!validTopics) {
+        navigate('/discussions');
+      }
```

Comment:
When user is updating the URL directly, I assume they are trying to find a certain topic for which they might know the exact name. I think showing a "not found" page would be much better here, lets confirm from product.

Reply:
@mzparacha Because the functionality of redirecting a user to All Discussions came from Dillon, I'm going to keep it as such.